lib/unifex/code_generators/tie_header.ex

```diff
@@ -0,0 +1,22 @@
+defmodule Unifex.CodeGenerators.TieHeader do
```

Comment:
`Unifex.CodeGenerators.*` namespace is intended for generators of interfaces, implementing `Unifex.CodeGenerator` behaviour. This should be `Unifex.CodeGenerator.TieHeader` or `generate_header` should be `Unifex.CodeGenerator.generate_tie_header`